### ISSUE TITLE
[Feature Request] Upcounting Counter

### ISSUE THREAD

**Issue by Darktimr** [ThemeFuse/Brizy#842]
Could you add a counter for since when something happened?

Like, the company was founded for 16 years 6 days.

Thanks for reply.

**Comment by bagrinsergiu** [comment]
Hi @Darktimr 

Counter element is available in left sidebar under elements tab: https://jmp.sh/9nKFjKi 

Can you confirm it  is what you need?

Sergiu

**Comment by Darktimr** [comment]
Hi @bagrinsergiu, i mean a counter that upcounts infinitely, for exampel, I input the date 05.03.2008 and the Counter shows me the year, days, Hours and Seconds since the date.

**Comment by bagrinsergiu** [comment]
Countdown?  https://jmp.sh/bFce0U2

**Comment by Darktimr** [comment]
Yes, but when i type in a date earlyer than 2018 than i will get 00:00:00

I want to see the difference between this two dates
